### ISSUE TITLE
Reference clients seem slightly outdated

### ISSUE THREAD

**Issue by robmoss2k** [ssllabs/ssllabs-scan#297]
A few very popular browsers are missing from Reference clients. Opera mobile browsers have around 4% of global browser market share. Opera desktop browsers have around 1%. Blackberry browsers have around 0.3%, but the major missing one is the UC Browser with a share of around 7%.

**Comment by ivanr** [comment]
It's possible. What is your source of information?

**Comment by abailey-wdc** [comment]
I agree that the list of reference clients should be cleaned up. I don't think a reference client should support any insecure protocol or ciphers out of the box, and needs to support TLS 1.2/1.3 with at least one FS cipher.

At the very least, these shouldn't be reference anymore:

- IE 8-10 / Win 7
- IE 11 / Win Phone 8.1
- IE 11 / Win Phone 8.1 Update
- Safari 6 / iOS 6.0.1
- Safari 6.0.4 / OS X 10.8.4
- Safari 7 / iOS 7.1
- Safari 7 / OS X 10.9
- Safari 8 / iOS 8.4
- Safari 8 / OS X 10.10
